cdk/lib/commercial-canaries.ts

```diff
@@ -86,15 +92,15 @@ export class CommercialCanaries extends GuStack {
 		});
 
 		new synthetics.CfnCanary(this, 'Canary', {
-			artifactS3Location: `s3://${S3Bucket}/canary/${awsRegion}/${canaryName}`,
+			artifactS3Location: `s3://${S3BucketResults}/${stage.toUpperCase()}`,
```

Comment:
It's unnecessary to make this upper case, but if the format of the stages is changed from being upper-case in the future for any reason, then this will not break because of it.